manifests/vanilla/vsphere-csi-driver.yaml

```diff
@@ -178,7 +175,7 @@ spec:
             - mountPath: /csi
               name: socket-dir
         - name: vsphere-csi-controller
-          image: gcr.io/cloud-provider-vsphere/csi/ci/driver:latest
+          image: gcr.io/cloud-provider-vsphere/csi/ci/driver:v2.3.0-rc.1
```

Comment:
use `gcr.io/cloud-provider-vsphere/csi/release/` instead of `gcr.io/cloud-provider-vsphere/csi/ci` at all places for the release version.

Reply:
done!